### ISSUE TITLE
NotepadNext-x86_64.AppImage 不能打开

### ISSUE THREAD

**Issue by falseUtopia** [dail8859/NotepadNext#335]
NotepadNext version [v0.6](https://github.com/dail8859/NotepadNext/releases/tag/v0.6)

Linux version 5.15.34-amd64-desktop 

```
qwer@asdf ~/app/NotepadNext  cat /etc/os-release 
PRETTY_NAME="Deepin 20.8"
NAME="Deepin"
VERSION_ID="20.8"
VERSION="20.8"
VERSION_CODENAME="apricot"
ID=Deepin
HOME_URL="https://www.deepin.org/"
BUG_REPORT_URL="https://bbs.deepin.org/"

qwer@asdf ~ldd --version
ldd (Debian GLIBC 2.28.21-1+deepin-1) 2.28


 ✘ qwer@asdf  ~/app/NotepadNext ./NotepadNext-x86_64.AppImage
/tmp/.mount_Notepaa3O4A2/AppRun.wrapped: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/.mount_Notepaa3O4A2/AppRun.wrapped)
/tmp/.mount_Notepaa3O4A2/AppRun.wrapped: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.30' not found (required by /tmp/.mount_Notepaa3O4A2/usr/bin/../lib/libsystemd.so.0)
```

我需要怎么处理？还是等待更新？

![image](https://user-images.githubusercontent.com/19901195/221338935-ffc474a5-45b4-4896-b80c-e335fd5ba9b5.png)

**Comment by falseUtopia** [comment]
#77

**Comment by dail8859** [comment]
Currently the GitHub action is using the oldest non-deprecated Linux version available (ubuntu-20.04). Qt6 binary packages are [linked against glibc 2.28](https://doc.qt.io/qt-6/supported-platforms.html#availability-of-packages), so it would still be possible to build and run it. However it is not something I'm willing to go out of my way to build it on a non-supported OS.
